### PR TITLE
Update overview.mdx

### PR DESCRIPTION
**Pull request by CharlesJonesFigment**
some rewording, cap changes.

### CODE REVIEW

File: docs/quickstart/overview.mdx [hunk 1]
```diff
@@ -26,7 +25,7 @@ All apps must have at least one protocol.
 
 ## Create an App
 
-**Step 2:** Give your App a name, select the environment - **Production** or **Staging** to differentiate between your apps' state, then select the protocol.
+**Step 2:** Give your app a name. Select the environment&mdash;**Production** or **Staging**&mdash;to differentiate between your app's state, then select the protocol.
```
Comment (dgamboa) [inline docs/quickstart/overview.mdx:28]: For this one, we should say:
**Step 2:** Give your app a name. Select the intended environment&mdash;**Production** or **Staging**&mdash;, then select the protocol.

File: docs/quickstart/overview.mdx [hunk 2]
```diff
@@ -70,7 +69,7 @@ All apps must have at least one protocol.
 
 ## Analytics
 
-**Step 6:** You can click on the **Analytics** tab to get access to your Request Logs. [Read more about Analytics and Request Logs](/quickstart/app-analytics-and-request-logs).
+**Step 6:** Select the **Analytics** tab to access to your request logs. [Read more about Analytics and Request Logs](/quickstart/app-analytics-and-request-logs).
```
Comment (dgamboa) [inline docs/quickstart/overview.mdx:72]: "...to access your..." rather than "...to your..."

File: docs/quickstart/overview.mdx [hunk 3]
```diff
@@ -81,7 +80,7 @@ All apps must have at least one protocol.
 
 ## Team Management
 
-**Step 7:** To invite your team members and collaborate with them on the project, click on the **Settings** tab, then click **Team** on the left side of the page. [Read more about App team roles and permissions](/quickstart/settings-and-security#adding-collaborators)
+**Step 7:** Invite your team members by selecting the **Settings** tab, then clicking **Team** on the left side of the page. [Read more about App team roles and permissions](/quickstart/settings-and-security#adding-collaborators)
```
Comment (dgamboa) [inline docs/quickstart/overview.mdx:83]: Should "App" in the link be "app" for consistency?

File: docs/quickstart/overview.mdx [hunk 4]
```diff
@@ -92,7 +91,7 @@ All apps must have at least one protocol.
 
 ## App Settings
 
-**Step 8:** To manage the payment method for your App, check invoices and delete apps, etc. click on the **Settings** tab. This is also where you can add CORS Allowed Origins, and manage your Team by adding or removing Collaborators.
+**Step 8:** Also under the **Settings** tab, you may manage your payment method and your app plan. You can also delete your app if you need to.
```
Comment (dgamboa) [inline docs/quickstart/overview.mdx:94]: "You can also add CORS allowed origins and delete your app if you need to."

I think we should keep the CORS piece.

**Review by dgamboa — CHANGES_REQUESTED** [review]
This is great. Added a couple of minor edits.

File: docs/quickstart/overview.mdx (hunk 1 above)
Comment (paulpomerleau) [reply docs/quickstart/overview.mdx]: updated

File: docs/quickstart/overview.mdx (hunk 2 above)
Comment (paulpomerleau) [reply docs/quickstart/overview.mdx]: updated

File: docs/quickstart/overview.mdx (hunk 3 above)
Comment (paulpomerleau) [reply docs/quickstart/overview.mdx]: updated

File: docs/quickstart/overview.mdx (hunk 4 above)
Comment (paulpomerleau) [reply docs/quickstart/overview.mdx]: updated

**Comment by paulpomerleau** [comment]
modified based on feedback, squashed and merged

**Pull request merged**